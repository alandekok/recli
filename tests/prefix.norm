(foo (bar|baz)|hello (a|b|there (me|you)))
