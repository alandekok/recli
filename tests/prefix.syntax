hello a
foo bar
hello b
foo baz
hello there you
hello there me
