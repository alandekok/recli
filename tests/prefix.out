hello
^ Parse error.
foo
^ Parse error.
